docs/se/guides/tracing.adoc

```diff
@@ -447,21 +466,19 @@ public class GreetService implements HttpService {
   private void outboundMessageHandler(ServerRequest request, ServerResponse response) {
     Invocation.Builder requestBuilder = webTarget.request();
 
-    // <2>
-    var spanBuilder = request.tracer()
-                .buildSpan("outboundMessageHandler");
-    request.spanContext().ifPresent(spanBuilder::asChildOf);
-    Span span = spanBuilder.start();
+    var spanBuilder = Tracer.global().spanBuilder("outboundMessageHandler");
+    request.context().get(SpanContext.class).ifPresent(sc -> sc.asParent(spanBuilder));
+    var span = spanBuilder.start();     // <2>
 
     try {
       requestBuilder.property(
-          ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, request.spanContext());  // <3>
+          ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, span.context());  // <3>
 
       String result = requestBuilder   // <4>
           .get(String.class);
       response.send(result);
```

Comment:
After the `response.send(result)` line add `span.end()` so in the normal case the span is properly ended.

Reply:
Fixed